pkg/machines/libvirt.es6

```diff
@@ -397,7 +397,7 @@ LIBVIRT_PROVIDER = {
                 metadata.installSource,
                 metadata.osVariant,
                 convertToUnit(currentMemory, units.KiB, units.MiB),
-                vcpus,
+                vcpus.count,
```

Comment:
Ouch! Good catch. However, doesn't this need to be a string? i.e . `vcpus.count.toString()`

Reply:
@martinpitt I ran for verification and there was not issue with parsing the int. If you prefer having all the values strings though I can change it.

Reply:
That's fine. Obviously our automatic tests don't cover this code path, otherwise we'd have noticed the crash earlier. So if you manually tested that, it's ok. Thanks!